fizzbuzz/java/mikegotta/MikesREADME.md

```diff
@@ -0,0 +1,13 @@
+Mike's FizzBuzz
+
+Assignment 3 
+-----------------------------------
+How to run Mikes FizzBuzz solution:
+- navigate to the directory fizzbuzz/java/mikegotta
+- change your permission with:  chmod +x runner.sh
```

Comment:
I think this should say runFizzBuzz.sh

Reply:
Hey Turner, I fixed that bug in the readme. As for the error you are getting... apparently it has something to do with java not being able to turn the string "17" into the integer "17". It works on my end, maybe I have a different version of java then you?